master/propidx.html

```diff
@@ -383,11 +381,7 @@ <h1>Property Index</h1>
         </tr>
         <tr>
           <th><a>'line-height'</a></th>
-          <td>normal |
-            <a href="https://www.w3.org/TR/2008/REC-CSS2-20080411/syndata.html#value-def-number">
-              <span class="value-inst-number noxref">&lt;number&gt;</span></a> |
-	    <a>&lt;length&gt;</a> |
-	    <a>&lt;percentage&gt;</a></td>
+          <td>normal | <a>&lt;number&gt;</a> | <a>&lt;length-percentage&gt;</a></td>
```

Comment:
Note: this is different from CSS definition of line-height: normal | <number> | <length> | <percentage> | inherit, but that is probably because CSS21 is still the latest definition.

Reply:
Yes, I don't think they are updating the syntax in CSS 2, and I can't find a level 3 definition of the property. But `calc(length + percentage)` works for `line-height`, so `<length-percentage>` is the correct unit type.